Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -264,9 +268,19 @@ open class AVFoundationPlayback: Playback {
     }
 
     private func seekToStartAtIfNeeded() {
-        if canStartAt {
-            seek(startAt)
-        }
+        guard canStartAt else { return }
+        
+        seek(startAt)
+    }
+    
+    private func seekToLiveStartTimeIfNeeded() {
```

Comment:
I think we are adding more than one responsibility in this method. An indication of that is the `ifNeeded` in the name.

Reply:
Done